osmoutils/accum/export_test.go

```diff
@@ -31,11 +31,15 @@ func (accum AccumulatorObject) GetAllPositions() ([]Record, error) {
 }
 
 // Creates an accumulator object for testing purposes
-func CreateRawAccumObject(store store.KVStore, name string, value sdk.DecCoins) AccumulatorObject {
+func CreateRawAccumObject(store store.KVStore, name string, value sdk.DecCoins, totalShares sdk.Dec) AccumulatorObject {
+	// We store an accumulator object in state even if unused in tests
+	// because position operations still require GetAccumulator to work
+	_ = MakeAccumulator(store, name)
```

Comment:
Since this PR makes position updates mutate accumulator state, it turns out that every call to this helper requires setting an accum in state.

I agree that changing the name would be better but every call to this function would require doing `MakeAccumulator` first, so having that in the helper avoids all that duplicate code